Server/Models/Item.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.Runtime.Serialization;
+
+namespace Karma.Server.Models
+{
+    [DataContract]
+    public class Item
+    {
+        [DataMember]
+        public string Name { get; set; }
+        [DataMember]
+        public string Description { get; set; }
```

Comment:
There should be `\n` between properties. For example:

[DataMember]
public string...

[DataMember]
public string...